portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +26,23 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private ArrayList<String> names;
 
+  public void init() {
```

Comment:
If you don't change this to use an immutable static list please add an `@Override` annotation.